source/Preferences.cpp

```diff
@@ -346,21 +357,21 @@ bool Preferences::ZoomViewOut()
 
 double Preferences::MinViewZoom()
 {
-	return ZOOMS[0];
+	return GameData::Interfaces().Get("hud")->GetList("zooms").front();
```

Comment:
Calling `front` on an empty vector is also undefined.
https://en.cppreference.com/w/cpp/container/vector/front